lib/classes/Variables.test.js

```diff
@@ -1594,10 +1594,19 @@ module.exports = {
           expect(getValueFromSsmStub).to.have.been.calledWith('ssm:/test/path/to/param');
         }));
 
-    it('should reject invalid sources', () =>
+    it('should reject invalid sources', () => {
       serverless.variables
         .getValueFromSource('weird:source')
-        .should.be.rejectedWith(serverless.classes.Error));
+        .should.be.rejectedWith(serverless.classes.Error);
```

Comment:
This returns promise, therefore for proper propagation it should be returned (I believe we need to wrap those three promises with `Promise.all` and return result)